if __name__ == "__main__":
    try:
        action()
    except Exception as e:
        print(f"Exception Caught: {str(e)}")